src/python/pants/backend/shell/target_types.py

```diff
@@ -430,13 +429,34 @@ class ShellCommandLogOutputField(BoolField):
 
 class ShellCommandWorkdirField(StringField):
     alias = "workdir"
-    default = None
+    default: ClassVar[Optional[str]] = "."
     help = softwrap(
-        "Sets the current working directory of the command, relative to the project root. If not "
-        "set, use the project root.\n\n"
+        "Sets the current working directory of the command. "
         "To specify the location of the `BUILD` file, use `.`. Values beginning with `.` are "
         "relative to the location of the `BUILD` file.\n\n"
-        "To specify the project/build root, use `/` or the empty string."
+        "To specify the build root, use `/` or the empty string.\n\n"
+        "Values that do not begin with `.` or `/` are relative to the build root."
+    )
+
+
+class RunShellCommandWorkdirField(ShellCommandWorkdirField):
+    default = None
+    help = softwrap(
+        "Sets the current working directory of the command that is `run`. If `None`, run the "
+        "command from the current working directory."
```

Comment:
In pants, I'm not sure if "current working directory" is very clear. Is it the project root, or the path of the BUILD file?

Also, with `scie-pants` this could be interpreted as the actual working directory from where you invoked `pants ..` which now may be != build root.

Reply:
Ah, I was trying to phrase that to match a `scie-pants` view of the world. It would be the _project root_ if using `./pants`, but probably now it's the directory from which `pants` is invoked.

Would: `run the command from the directory where you invoke Pants` make more sense?